src/editor/utils/elementapimixin.js

```diff
@@ -19,8 +19,8 @@ const ElementApiMixin = {
 	/**
 	 * @inheritDoc
 	 */
-	updateElement() {
```

Comment:
Let's add a check here that `sourceElement` exists and throw otherwise. We expect that it may not be set.

Reply:
Done in e8cc7ed.